example/src/main/java/io/netty/example/http/websocketx/benchmarkserver/WebSocketServerHandler.java

```diff
@@ -115,21 +115,21 @@ private void handleWebSocketFrame(ChannelHandlerContext ctx, WebSocketFrame fram
 
         // Check for closing frame
         if (frame instanceof CloseWebSocketFrame) {
-            handshaker.close(ctx, (CloseWebSocketFrame) frame.retain());
+            handshaker.close(ctx, (CloseWebSocketFrame) frame);
```

Comment:
Like with the MTMD, the SimpleChannelInboundHandler releases the messages for us, so not having these retains here just happens to work out because neither of them have been ported to Buffer.